src/algebra/order/archimedean.lean

```diff
@@ -276,6 +276,30 @@ archimedean_iff_nat_lt.trans
  λ H x, let ⟨n, h⟩ := H x in ⟨n+1,
    lt_of_le_of_lt h (nat.cast_lt.2 (lt_add_one _))⟩⟩
 
+lemma archimedean_iff_int_lt : archimedean α ↔ ∀ x : α, ∃ n : ℤ, x < n :=
+begin
+  rw archimedean_iff_nat_lt,
+  refine ⟨λ h x, _, λ h x, _⟩,
+  { rcases h x with ⟨n, h⟩,
+    refine ⟨n, _⟩,
+    exact_mod_cast h },
+  { rcases h x with ⟨n, h⟩,
+    refine ⟨n.to_nat, h.trans_le _⟩,
+    exact_mod_cast int.le_to_nat _ }
+end
```

Comment:
If we bring `exists_int_gt` in line with `exists_nat_gt` by changing its hypothesis from
`[linear_ordered_ring α] [archimedean α]` to `[nontrivial α] [ordered_ring α] [archimedean α]` then we can write this as:
```suggestion
⟨@exists_int_gt α _ _,
begin
  rw archimedean_iff_nat_lt,
  intros h x,
  obtain ⟨n, h⟩ := h x,
  refine ⟨n.to_nat, h.trans_le _⟩,
  exact_mod_cast int.le_to_nat _,
end⟩
```